src/entities/AppUser.ts

```diff
@@ -36,7 +36,8 @@ export class AppUser {
   @Column({ nullable: true })
   graduationYear: string;
 
-  @ManyToOne(() => InductionClass, { nullable: true })
+  @ManyToOne(() => InductionClass, { nullable: true, eager: true })
+  @JoinTable()
```

Comment:
why do we need a join table here? it's many to one not many to many

Reply:
I thought Join table would tell typeorm to get the data from the other table as well instead of just returning the inductionClass value.

Reply:
doesn't eager do the job? join table creates a new table in the db

Reply:
You're right. Removed JoinTable.